core/commands/pin.go

```diff
@@ -166,9 +189,18 @@ var addPinCmd = &cmds.Command{
 				if !ok {
 					return e.TypeErr(out, v)
 				}
-				if out.Pins == nil {
-					// this can only happen if the progress option is set
-					fmt.Fprintf(os.Stderr, "Fetched/Processed %d nodes\r", out.Progress)
+
+				// this can only happen if the progress/verbose options are set
+				if inProgress := out.Pins == nil; inProgress {
+					if showVerboseProgress {
+						for _, pin := range out.OngoingPins {
+							fmt.Fprintf(os.Stderr,
+								"Pinning %s\n", pin)
+						}
+					} else if showDigitProgress {
```

Comment:
Should we also include the numeric progress while in verbose mode?

I'm kind of wondering whether there should really be separate `verbose` and `progress` modes. Maybe this work you're doing should expand the meaning of the `progress` flag instead of introducing a new one? cc @magik6k/@Stebalien do either of you have an opinion on whether or not we expand the progress flag to show more infomation? vs introducing a new flag for showing more verbose progress?